### PR TITLE
mediatek: add ABT ASR3000 support

### PR DESCRIPTION
**Pull request by 1715173329**
```
Hardware specification:
  SoC: MediaTek MT7981B 2x A53
  Flash: 128 MB SPI-NAND
  RAM: 256MB
  Ethernet: 4x 10/100/1000 Mbps
  Switch: MediaTek MT7531AE
  WiFi: MediaTek MT7976C
  Button: Reset, Mesh
  Power: DC 12V 1A

Gain telnet access:
1. Login into web interface, and download the configuration.
2. Decode and uncompress the configuration:
     * Enter fakeroot if you are not login as root.
   base64 -d e-xxxxxxxxxxxx-cfg.tar.gz | tar -zx
3. Edit 'etc/passwd', remove root password: 'root::1:0:99999:7:::'.
4. Edit 'etc/rc.local', insert telnetd command before 'exit 0':
   ( sleep 3s; /usr/sbin/telnetd; ) &
5. Repack the configuration:
   tar -zc etc/ | base64 > e-xxxxxxxxxxxx-cfg.tar.gz
6. Upload new configuration via web interface, now you can connect to
   ASR3000 via telnet.

Flash instructions:
1. Connect to ASR3000, backup everything, especially 'Factory' part.
2. Write new BL2:
   mtd write openwrt-mediatek-filogic-abt_asr3000-preloader.bin BL2
3. Write new FIP:
   mtd write openwrt-mediatek-filogic-abt_asr3000-bl31-uboot.fip FIP
4. Set static IP on your PC:
   IP 192.168.1.254/24, GW 192.168.1.1
5. Serve OpenWrt initramfs image using TFTP server.
6. Cut off the power and re-engage, wait for TFTP recovery to complete.
7. After OpenWrt has booted, perform sysupgrade.
```

### CODE REVIEW

**Comment by 1715173329** [comment]
@dangowrt @hauke could you have a look please?

File: target/linux/mediatek/dts/mt7981b-abt-asr3000.dts [hunk 1]
```diff
@@ -0,0 +1,275 @@
...
+
+		mesh_led: led-2 {
+			label = "green:mesh";
+			gpios = <&pio 15 GPIO_ACTIVE_HIGH>;
+		};
+
+		led-3 {
+			function = LED_FUNCTION_WLAN_2GHZ;
+			color = <LED_COLOR_ID_GREEN>;
+			gpios = <&pio 34 GPIO_ACTIVE_LOW>;
```
Comment (hauke) [inline target/linux/mediatek/dts/mt7981b-abt-asr3000.dts:70]: Is it working to define the trigegr in device tree?
```
linux,default-trigger = "phy0tpt";
```
or
```
linux,default-trigger = "phy0radio";
```

Comment (hauke) [reply target/linux/mediatek/dts/mt7981b-abt-asr3000.dts:70]: Just asked ChatGPT for the difference between the two options:
> phy0tpt: LED indicates transmission activity (i.e., when packets are being sent).
> phy0radio: LED indicates the state of the radio transmitter (i.e., whether the radio is on or off).

Comment (1715173329) [reply target/linux/mediatek/dts/mt7981b-abt-asr3000.dts:70]: I've tested them but both triggers were not working properly.
phy\*radio just turned the LED off.
phy\*tpt made the LED blink at a certain frequency but did not indicate transmission activity.

Comment (hauke) [reply target/linux/mediatek/dts/mt7981b-abt-asr3000.dts:70]: ok strange. then just use the current code.

**Pull request merged**